kani-driver/src/args/mod.rs

```diff
@@ -252,6 +252,14 @@ pub struct VerificationArgs {
     #[arg(long, hide_short_help = true, requires("enable_unstable"))]
     pub ignore_global_asm: bool,
 
+    /// Ignore lifetimes of local variables. This effectively extends their
+    /// lifetimes to the function scope, and hence may cause Kani to miss
+    /// undefined behavior resulting from using the variable after it dies.
+    /// This option may impact the soundness of the analysis and may cause false
+    /// proofs and/or counterexamples
+    #[arg(long, hide_short_help = true, requires("enable_unstable"))]
```

Comment:
we shouldn't use `--enable-unstable` anymore

Reply:
Instead, can you convert this flag into `-Z ignore_locals_lifetime`?

Reply:
I went back and forth on this one. The -Z flags seemed to me more like unstable features. Since this flag isn't really a feature, but a workaround for a bug, -Z didn't seem appropriate to me. I saw that https://github.com/model-checking/kani/issues/3068 mentioned adding a -Z unstable-options that can be used for such cases, so I thought I'd use enable-unstable for the time being. What do you think?

Reply:
Resolving for the time being so that this PR can make it into the release. @celinval @zhassan-aws please follow-up if still needed.